internal/engine/upper.go

```diff
@@ -405,6 +416,32 @@ func handleGlobalYAMLApplyError(
 	state.GlobalYAMLState.LastError = event.Error
 }
 
+func handleTiltfileReloaded(
+	ctx context.Context,
+	state *store.EngineState,
+	event TiltfileReloadedAction,
+) {
+	manifests := event.Manifests
+	globalYAML := event.GlobalYAML
+	err := event.Err
+	if err != nil {
+		logger.Get(ctx).Infof("Unable to parse Tiltfile: %v", err)
```

Comment:
This probably needs to return after the error?

Can you add a test for this case?

Reply:
Also: where does this error appear? We probably want to print the backtrace

Reply:
Right now this error just appears in the logs (`L` in the UI).

I don't think we want to return here. I think if you save the Tiltfile with a syntax error `tilt` itself shouldn't stop or crash. What do you think?

Reply:
i don't think we're disagreeing?

if you introduce a syntax error in the Tiltfile, we should keep displaying the services that are there. 

right now, you've changed it so that it will delete all the services by setting the list of manifests to empty

i'm suggesting that you return here, skipping the codepath below, so that it doesn't reset the manifests.

i'm happy to write a unit test for the expected behavior if it's unclear.

Reply:
Ohhh I thought if you `return`ed here it would stop everything. Great, will do that with a test!

Reply:
Actually I've changed the behavior so that if a rebuild of a manifest fails we set the `LastManifestLoadError` to the Tiltfile execution error on each manifest state. This matches the current behavior, and displays alongside each manifest in the HUD.